scrapper.py

```diff
@@ -26,38 +36,123 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        class_bs = article_bs.find('div', class_="b-materials-list b-list_infinity")
+        title_bs = class_bs.find_all('p', class_="b-materials-list__title b-materials-list__title_news")
+        for link in title_bs:
+            link = link.find('a')
+            self.urls.append('https://www.m24.ru' + link['href'])
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        software_names = [SoftwareName.CHROME.value]
+        operating_systems = [OperatingSystem.WINDOWS.value, OperatingSystem.LINUX.value]
+        useragent_random = UserAgent(software_names=software_names, operating_systems=operating_systems, limit=100)
+        headers = {'user-agent': useragent_random.get_random_user_agent(),
+                   'accept': '*/*', 'accept-language': 'ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7'}
+
+        for seed_url in self.seed_urls:
+            response = requests.get(seed_url, headers=headers, allow_redirects=True, timeout=5)
+            print(response.status_code)
+            with open('index1.html', 'w', encoding='utf-8') as file:
+                file.write(response.text)
```

Comment:
This code is going to create a html file in the directory where scrapper.py is located. 
We do not need to save all the visited pages as html.
Refer to `docs/datset.md` to see where and how data must be stored.

Reply:
okay, I fixed that

Reply:
as far as I can see, it is still there

Reply:
sorry, now I deleted these strings
all files stored here: 2021-1-level-ctlr\tmp\articles